ethsigner/src/main/java/tech/pegasys/ethsigner/jsonrpc/SendTransactionJsonParameters.java

```diff
@@ -29,22 +29,20 @@
   private static final int HEXADECIMAL = 16;
   private static final int HEXADECIMAL_PREFIX_LENGTH = 2;
 
-  private final String data;
   private String sender;
 
   private BigInteger gas;
   private BigInteger gasPrice;
   private BigInteger nonce;
   private BigInteger value;
   private String receiver;
+  private String data;
 
   @JsonCreator
-  public SendTransactionJsonParameters(
-      @JsonProperty("from") final String sender, @JsonProperty("data") final String data) {
-    this.data = data;
-
+  public SendTransactionJsonParameters(@JsonProperty("from") final String sender) {
     validatePrefix(sender);
     this.sender = sender;
+    this.data = "";
```

Comment:
do we want to catch this here, or in the "Converter" where we insert the default values?

Reply:
Moved to the Converter